eslint-config/index.mjs

```diff
@@ -102,10 +101,21 @@ export default tseslint.config(
 		rules: {
 			'import/order': ['warn', {
 				'groups': ['builtin', 'external', 'internal', 'parent', 'sibling', 'index', 'type'],
-				'newlines-between': 'never'
+				'newlines-between': 'never',
+				'sortTypesGroup': true
 			}],
 			'import/first': 'error',
-			'import/consistent-type-specifier-style': ['error', 'prefer-top-level']
+			'import/consistent-type-specifier-style': ['error', 'prefer-top-level'],
+			'import/no-relative-packages': 'warn',
+			'import/no-relative-parent-imports': 'warn',
```

Comment:
As discussed in Discord, this rule should be removed as it fails for type aliases

<img width="1639" height="151" alt="Image" src="https://github.com/user-attachments/assets/41d249a8-d600-4b3a-859e-d844d0527b64" />

The `no-restricted-imports` will cover the relative imports being used

Reply:
Do you want me to remove both the `no-relative` rules or just the `parent-imports` one? We only discussed the `parent-imports` one on Discord but now I'm wondering if the other one needs to be there either